src/compat/eme/custom_media_keys/webkit_media_keys.ts

```diff
@@ -219,9 +232,22 @@ export default function getWebKitMediaKeysCallbacks() {
     throw new Error("No WebKitMediaKeys API.");
   }
   const isTypeSupported = WebKitMediaKeysConstructor.isTypeSupported;
-  const createCustomMediaKeys = (keyType: string) => new WebKitCustomMediaKeys(keyType);
+  const createCustomMediaKeys = (keyType: string) =>
+    new WebKitCustomMediaKeys(keyType);
+  const setMediaKeys = (elt: HTMLMediaElement,
+                        mediaKeys: MediaKeys|ICustomMediaKeys|null): void => {
+    if (mediaKeys === null) {
+      return setWebKitMediaKeys(elt, mediaKeys);
+    }
+    if (mediaKeys !== null && !isCustomMediaKeys(mediaKeys)) {
```

Comment:
First, this condition cannot be true here (due to the first check).
Second, as usual, why not check about if `mediaKeys` is a `WebKitCustomMediaKeys` instead?